doc/swss-schema.md

```diff
@@ -106,7 +106,7 @@ For example (reorder output)
     oper_status         = "down" / "up"        ; operating status
     mtu                 = 1*4DIGIT             ; MTU for the IP interface of the VLAN
 
-    key                 = VLAN_TABLE:vlanid:ifname ; physical port member of VLAN
+    key                 = VLAN_MEMBER_TABLE:vlanid:ifname ; physical port member of VLAN
```

Comment:
can you add a separate table here?

For the VLAN_MEMBER_TABLE, tagging_mode is its attribute. 

For the VLAN_TABLE, it does not need to have tagging_mode as an attribute.